app/src/views/PoolPage/usePoolPageData.tsx

```diff
@@ -180,9 +180,11 @@ export const usePoolPageData = () => {
           );
 
         const pool = useCore().store.pools[poolKey];
-
-        const denomOrSymbol =
-          pool.externalAmount.ibcDenom ?? pool.externalAmount.symbol;
+        const ibcDenom =
+          tokenRegistryEntriesQuery.data.value?.registry?.entries.find(
+            (y) => y.baseDenom === pool.externalAmount.symbol,
+          )?.denom;
+        const denomOrSymbol = ibcDenom ?? pool.externalAmount.symbol;
```

Comment:
Do you mind explaining this? thanks

Reply:
I needed to introduce this code to get ibc hash of external token. For some reason it wasn't available as part of `pool.externalAmount.ibcDenom`